vic/drivers/cesm/bld/vic.globalconfig.txt

```diff
@@ -67,16 +67,21 @@ OUTVAR      OUT_BASEFLOW
 OUTVAR      OUT_SWE
 OUTVAR      OUT_EVAP
 OUTVAR      OUT_SOIL_TEMP
-
-OUTFILE     {caseid}.vic.hmm
-AGGFREQ     NMONTHS 1
-OUT_FORMAT  NETCDF4_CLASSIC
-HISTFREQ    NMONTHS 1
-OUTVAR      OUT_AIR_TEMP
```

Comment:
Don't you need to specify
```
OUTFILE
AGGFREQ
OUT_FORMAT
HIST_FREQ
```
anymore?

Reply:
They still are specified for the daily output file, I just removed the monthly output file entirely

Reply:
they are still specified for the daily output file, I just removed the monthly output file entirely